src/gru.js

```diff
@@ -2,12 +2,13 @@
 
 import {concat} from './concat.js';
 import {gruCell} from './gru_cell.js';
-import {reshape} from './reshape.js';
-import {sizeOfShape, Tensor} from './tensor.js';
-import {sigmoid} from './sigmoid.js';
+import {reshape} from './lib/reshape.js';
+import {sizeOfShape, Tensor} from './lib/tensor.js';
+import {sigmoid} from './lib/sigmoid.js';
```

Comment:
`sigmoid` is a WebNN op: https://www.w3.org/TR/webnn/#api-mlgraphbuilder-sigmoid